bundle-workflow/src/test_workflow/test_result/test_result.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+
+class TestResult:
+    def __init__(self, component, config, status):
+        self.component = component
+        self.config = config
+        self.status = status
+
+    @property
+    def __test_result(self):
+        return "PASS" if self.status == 0 else "FAIL"
+
+    def __str__(self):
+        return f"{self.__test_result} Test for {self.component} {self.config} with status code {self.status}"
```

Comment:
nit: if we are logging this for summary, it might not be very readable. Can we have something tabular, e.g. 


| index-management  |  without-security | SUCCESS |  
| index-management  |  with-security | FAILURE |
| anomaly-detection  |  without-security | SUCCESS |